packages/js-moi-providers/src.ts/base-provider.ts

```diff
@@ -1031,6 +1098,45 @@ export class BaseProvider extends AbstractProvider {
         }
     }
 
+    public async getSubscription(event: ProviderEvents): Promise<string> {
+        let params: unknown = [];
+
+        if (typeof event === "string") {
+            params = event;
+        }
+
+        if (typeof event === "object") {
+            if (event.event === 'newTesseractsByAccount') {
+                if (!isValidAddress(event.params)) {
+                    ErrorUtils.throwArgumentError("Invalid address provided", "event.params", event.params);
+                }
+
+                params = [event.event, { address: event.params }];
```

Comment:
Make params as object. Don't mix between string and object.

Reply:
https://github.com/sarvalabs/js-moi-sdk/pull/88/commits/4f623493fa8b42c40441616b75d5a2980762a75f